src/tests/test_stream.cpp

```diff
@@ -39,7 +40,12 @@ class Stream_Cipher_Tests : public Text_Based_Test
 
          for(auto&& provider: providers)
             {
-            std::unique_ptr<Botan::StreamCipher> cipher(Botan::StreamCipher::create(algo, provider));
+            std::unique_ptr<Botan::StreamCipher> cipher;
+            try {
+               cipher = Botan::StreamCipher::create(algo, provider);
```

Comment:
Remove please, just leaving the changes to adding the Seek param. Other than that this looks ready to merge.